app/javascript/widget/components/template/IntegrationCard.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div>
+    <button
+      class="button join-call-button"
+      color-scheme="secondary"
+      :is-loading="isLoading"
+      :style="{
+        background: widgetColor,
+        borderColor: widgetColor,
+        color: textColor,
+      }"
```

Comment:
This is how it is done in most places where we had to customize the colors in the widget based on the user-selected widget brand color. With the latest tailwind, I believe we could do `text-${textColor}` and `bg-${widgetColor}`. However, I don't think this works with the version we are using.

Reply:
> I don't think this works with the version we are using.

Yeah it does not, the processor is static, the hacky way to do this is having all possible combinations of this in a comment some where

> With the latest tailwind, I believe we could do text-${textColor} and bg-${widgetColor}

Custom properties are allowed like "bg-[#ff22ee]" but the processing is still static in nature, so having JS templates won't work

Reply:
In this case, the computed methods can simply generate the CSS class names, which can then be attached to the `class` property, instead of being attached to `style`

In case we need to use something like `text-${textColor}` we can add it to the `allowList` / `whiteList` in the purgeCSS options in the tailwind config

Reply:
I'm not sure if I understand. We cannot add an allow list of the classes as the color is selected by the user, we don't have control over what color they would choose.